packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0
+
+* Endorses `package:google_maps_flutter_web` as the web implementation of this plugin.
```

Comment:
Nit: we usually just use the package name, not `package:`, when referring to a package.

Reply:
Removed "package:", and added a link to the package in pub, to maximize the chances that people find its readme :)